internals/secrethub/read.go

```diff
@@ -36,6 +37,7 @@ func NewReadCommand(io ui.IO, newClient newClientFunc) *ReadCommand {
 		clearClipboardAfter: defaultClearClipboardAfter,
 		io:                  io,
 		newClient:           newClient,
+		newWriter:           ioutil.WriteFile,
```

Comment:
I think the name `newWriter` does not reflect what the function stored in this variable does.

If a function's name starts with `new` it implies that it is a constructor for something, in this case a writer, so I would expect it to return an `io.Writer` and take all parameters necessary to construct a writer.

In this case the function writes some data to a file, so I'd name it something like `writeFileFunc`. We have used the `Func`  suffix to denote that a variable is a function in situations like this.

Reply:
Fixed